docs/source/iroha_2_whitepaper.md

```diff
@@ -202,8 +202,7 @@ the submitting peer can use this as proof to convince non-faulty nodes to do a v
 
 - after a node signs off on a block and forwards it to the *proxy tail*,
 they expect a commit message within a reasonable amount of time (the *commit time*);
-if there is no commit message in time, the node tries to convince the network to do a view change,
-which creates a new leader and proxy tail by shifting down the 2*f*+1 nodes used by 1 to the right
+if there is no commit message in time, the node starts suspecting the *a* set.
```

Comment:
```suggestion
if there is no commit message in time, the node starts suspecting the $a$ set.
```

Reply:
We're essentially differentiating between mathematical and non-mathematical usage of terms. Believe me, it'll render correctly.